includes/modules/attributes.php

```diff
@@ -540,10 +541,11 @@
 
 
         // default
-        // find default attribute if set for default dropdown
+        // find default attribute if set. Intended for dropdown's default
         if ($products_options->fields['attributes_default'] == '1') {
-            $selected_attribute = $products_options_value_id;
+            $selected_select_attribute = $products_options_value_id;
         }
+        $selected_attribute = $selected_select_attribute;

```

Comment:
Just to be sure I understand: the outcome of this code is identical to the original, except that another (presently unused) variable is now also at play, right?